harvard-lts/CURIOSity
# Gallery Focus State

**Gallery Focus State**
* * *

**JIRA Ticket**: [LTSMAINT-764](https://jira.huit.harvard.edu/browse/LTSMAINT-764)

# What does this Pull Request do?
Adds a visual outline to gallery card when focus on title link, so you can visually see that the image and title are associated together

# How should this be tested?
* Spin up Curiosity from this branch
* Navigate to any exhibit's search results and click on the gallery view
* Begin tabbing through the page
* When you get to the items in the search results, an outline will appear around both the image AND title (almost like you're selecting a card); you should only see this behavior in the gallery view, not list or masonry

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@VanessaVenti @enriquediaz
